src/prefect_server/api/deprecated/flows.py

```diff
@@ -0,0 +1,172 @@
+from prefect import api
+from prefect_server.database import models
+from prefect.utilities.plugins import register_api
+
+
+@register_api("flows._update_flow_setting")
+async def _update_flow_setting(flow_id: str, key: str, value: any) -> models.FlowGroup:
```

Comment:
Not important since these are deprecated anyway, but why is this "private"?

Reply:
Historic artifact from an earlier attempt at the server API. I will update the new version in `/api/flow_groups` (but leave this one for compat)